### DISCUSSION TITLE
Looking for maintainer

### DISCUSSION THREAD

**Discussion by kjharcombe** [SilentVoid13/Templater#771] [Help]
I noticed in the Obsidian Roundup that you are looking for a maintainer for Templater. I'd like to volunteer to take that on if it would be helpful to you.

**Comment by AB1908** [comment]
I'd love to chip in occasionally to help out. I try to keep my eye on issues to see if I can provide answers and Welpdx does as well. We'll probably continue doing so to keep alleviating some of the lifting here.

**Comment by welpdx** [comment]
@SilentVoid13 If you only want one maintainer for now, I recommend @AB1908. He's been keeping on top of things here and discord. 

I mainly just respond to issues and discussions only.

**Comment by AB1908** [comment]
Haha I can't take it on. Would much rather help a separate person stay on top of things 💜

**Comment by Aithr-13** [comment]
Saw in the roundup you were looking for someone to help do maintaining. 

Junior dev, not a ton of JS/Web technologies experience - I do C++ and Python at work. Let me know if that would be helpful, or not.

**Comment by micseydel** [comment]
In https://github.com/SilentVoid13/Templater/discussions/702 I quoted
> Based on things others have said in Discord, I suspect that even if you're not a coder, just helping consolidate and categorize issues helps developers save a lot of time.

and today someone said
> I think the new maintainer needs to be reasonably familiar with the codebase and have sufficient spare time to address merges/fixes.

Is there any centralized place for this discussion? What the requirements/non-requirements of a new maintainer would be?

Is there somewhere I could start? I see 128 open issues but I have no idea the relative priority or difficulty.

**Comment by AB1908** [comment]
This is the discussion. I think my points from earlier about having free time and being reasonably familiar with the codebase still stand though the latter isn't that important a requirement I guess.

**Comment by micseydel** [comment]
Alright, if someone wants to point me toward something I can make a PR for, that would be cool. If not... 🤷

**Comment by AB1908** [comment]
It's really not necessary that you make a PR to become a maintainer. If you're just looking to contribute, a good start would be issues with several comments. There's one open issue that I've not found the time for which is a good start. #691 is the one.

Being a maintainer comes with the responsibility of actively triaging issues and reading PRs and giving feedback. Needs some familiarity with the Obsidian API as well. As of now, the three main maintainers I've seen are shabegom, Liam and SilentVoid. Welpdx and I don't really have collaborator access but we try our best to triage issues and get replies to people so that they can at least have a response. Welpdx often does more of the heavy lifting here and I'd love to do more but it's a real requirement that one needs free time.

**Comment by AB1908** [comment]
Howdy folks, is anyone still interested in being a maintainer? We still need some help staying on top of discussions, issues, triaging, etc.

**Reply by kjharcombe** [reply]
@AB1908 I would be

**Reply by Evelf** [reply]
Me too!

**Reply by AB1908** [reply]
Oh I meant to reply that please start helping out haha. The plugin could use some love!

This isn't to say that you are required to help before being a maintainer but at least we can start working together figure that part out.
